src/query/sql/src/executor/physical_plans/physical_aggregate_partial.rs

```diff
@@ -41,29 +42,55 @@ pub struct AggregatePartial {
 impl AggregatePartial {
     pub fn output_schema(&self) -> Result<DataSchemaRef> {
         let input_schema = self.input.output_schema()?;
-        let mut fields =
-            Vec::with_capacity(self.agg_funcs.len() + self.group_by.is_empty() as usize);
+        // let mut fields =
```

Comment:
Incorrect changes if we are using old hashtable.

Better to put a seting inside the plan struct.